api/src/controllers/referent.js

```diff
@@ -540,6 +545,21 @@ router.put("/young/:id", passport.authenticate("referent", { session: false, fai
       newYoung.populationDensity = populationDensity;
     }
 
+    // when changing statusPhase2, we check applications and equivalence
+    if (newYoung.statusPhase2 && newYoung.statusPhase2 === YOUNG_STATUS_PHASE2.VALIDATED) {
+      const applications = await ApplicationModel.find({ youngId: young._id });
+      const pendingApplication = applications.filter((a) => [APPLICATION_STATUS.WAITING_VALIDATION, APPLICATION_STATUS.WAITING_VERIFICATION].includes(a.status));
+
+      const equivalence = await MissionEquivalenceModel.find({ youngId: young._id });
+      const pendingEquivalence = equivalence.filter((e) => [EQUIVALENCE_STATUS.WAITING_CORRECTION, EQUIVALENCE_STATUS.WAITING_VERIFICATION].includes(e.status));
```

Comment:
```suggestion
      const pendingEquivalence = equivalences.filter((equivalence) => [EQUIVALENCE_STATUS.WAITING_CORRECTION, EQUIVALENCE_STATUS.WAITING_VERIFICATION].includes(equivalence.status));
```

ou `equ` dans le filter, c'est juste pour que ça soit plus lisible, quand je vois `e` je pense "erreur"